packages/atomic/src/components/search-box-suggestions/atomic-search-box-instant-results/atomic-search-box-instant-results.tsx

```diff
@@ -66,8 +66,8 @@ export class AtomicSearchBoxInstantResults {
     const results = this.instantResults.state.results.length
       ? this.instantResults.state.results
       : this.results;
-    return results.map((result: Result) => ({
-      key: `instant-result-${cleanUpString(result.title)}`,
+    return results.map((result: Result, i: number) => ({
+      key: `instant-result-${cleanUpString(result.title)}-${i}`,
```

Comment:
This isn't recommended with JSX. Instead of using the result.title, we should use the result `uniqueId` and perhaps remove the need for `cleanUpString`

> Do not use the map-function's index variable as a key. It does not represent a stable identity of an item as it can change if the order of the list changed or if you added an item to the beginning of the list. As such it is not suitable as a key.

https://stenciljs.com/docs/templating-jsx#using-jsx

Reply:
yeah, the problem with the unique ID is that it's got weird characters and it was breaking everything... but I guess I could clean that one up instead! alright, will update!